x-pack/plugins/security_solution/public/resolver/view/panels/cube_for_process.tsx

```diff
@@ -11,48 +11,73 @@ import { i18n } from '@kbn/i18n';
 /* eslint-disable react/display-name */
 
 import React, { memo } from 'react';
-import { useResolverTheme } from '../assets';
+import { useResolverTheme, SymbolIds } from '../assets';
+
+interface StyledSVGCube {
+  readonly isOrigin?: boolean;
+}
 
 /**
  * Icon representing a process node.
  */
 export const CubeForProcess = memo(function ({
+  className,
   running,
+  isOrigin,
   'data-test-subj': dataTestSubj,
 }: {
   'data-test-subj'?: string;
   /**
    * True if the process represented by the node is still running.
    */
   running: boolean;
+  isOrigin?: boolean;
+  className?: string;
 }) {
   const { cubeAssetsForNode } = useResolverTheme();
-  const { cubeSymbol } = cubeAssetsForNode(!running, false);
+  const { cubeSymbol, strokeColor } = cubeAssetsForNode(!running, false);
 
   return (
-    <StyledSVG width="1.5em" height="1.5em" viewBox="0 0 1 1" data-test-subj={dataTestSubj}>
+    <StyledSVG
+      className={className}
+      width="2.15em"
+      height="2.15em"
+      viewBox="0 0 100% 100%"
+      data-test-subj={dataTestSubj}
+      isOrigin={isOrigin}
+    >
       <desc>
         {i18n.translate('xpack.securitySolution.resolver.node_icon', {
           defaultMessage: '{running, select, true {Running Process} false {Terminated Process}}',
```

Comment:
:grey_question: Do we need to update this because it could also be "Analyzed Event"? Maybe not because it's already in the text below?

Reply:
No, shouldn't be necessary because it's in the text below, but also because the process/icon is still of type `running` or `terminated`